packages/lesswrong/server/scripts/generativeModels/generateTaggingPostSets.ts

```diff
@@ -56,7 +56,7 @@ async function generateCandidateSetsForTagClassification(): Promise<void> {
     draft: false, status: postStatuses.STATUS_APPROVED,
     isFuture: false, unlisted: false, shortform: false, authorIsUnreviewed: false,
     question: false, isEvent: false,
-    contents: {$ne: null},
+    // contents: {$ne: null},
```

Comment:
It does need to be uncommented for the script to run now (because of contents being normalised), but in general I haven't brought these scripts up to the present, so I think I'll uncomment it but leave a TODO

Reply:
Btw is there a new canonical way of selecting based on the value of a joined field, or should this use a repo function?